internal/guest/storage/devicemapper/devicemapper.go

```diff
@@ -216,6 +219,43 @@ func makeTableIoctl(name string, targets []Target) *dmIoctl {
 	return d
 }
 
+// CreateDeviceWithRetryErrors keeps retrying to create device mapper target
+func CreateDeviceWithRetryErrors(
+	ctx context.Context,
+	name string,
+	flags CreateFlags,
+	targets []Target,
+	errs ...error,
+) (string, error) {
+retry:
+	for {
+		dmPath, err := CreateDevice(name, flags, targets)
+		if err == nil {
+			return dmPath, nil
+		}
+		log.G(ctx).WithError(err).Error("CreateDevice error")
```

Comment:
Is this an error if its retriable? Would it make sense to log as a Warn to keep it separate from unrecoverable issues?